osu.Game/Overlays/Mods/ModPanel.cs

```diff
@@ -70,7 +70,7 @@ public ModPanel(ModState modState)
             Content.Masking = true;
             Content.CornerRadius = CORNER_RADIUS;
             Content.BorderThickness = 2;
-            Content.Shear = new Vector2(ShearedOverlayContainer.SHEAR, 0);
+            Shear = new Vector2(ShearedOverlayContainer.SHEAR, 0);
```

Comment:
Probably wanna do the same for `ShearedButton` which is also suffering from this issue, maybe another PR.

Reply:
Yep, it's a bit more involved as padding is applied to it (causing the visual area to shrink but not the clicking area). Removing it will cause adverse effects (buttons will expand, the spacing will be different, etc).